### DISCUSSION TITLE
celltype SC-ST mapping

### DISCUSSION THREAD

**Discussion by JingGuo1997** [BayraktarLab/cell2location#267] [Q&A]
Hello developers of cell2location,
 
 When I used cell type mapping from single cell data to space, I found that too many spots were predicted as one celltype, but in fact, the number of such cell types in tissues is very small. Can I change  parameters and adjust the cell type annotation to  match biology？

  We look forward to your reply

**Comment by vitkl** [comment]
Hi @JingGuo1997 

I don't understand what you mean. The model predicts the abundance of cell types: one "spot" can contain one or multiple cell types in varying amounts.
